digitaltwins/samples/model_repository/create_model.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+const ModelRepositoryCredentials = require('azure-iot-digitaltwins-model-repository').ModelRepositoryCredentials;
+const ModelRepositoryServiceClient = require('azure-iot-digitaltwins-model-repository').ModelRepositoryServiceClient;
+const uuid = require('uuid');
+
+function createInterfaceDocument() {
+  interfaceSample = {
```

Comment:
Should we read digitaltwins/samples/model_repository/sample_interface.json from the file instead of having a duplicate and inlined copy here?

Reply:
We could, but I wanted to have a self contained sample.
What do you think?

---
In reply to: [338374759](https://github.com/Azure/azure-iot-sdk-node/pull/646#discussion_r338374759) [](ancestors = 338374759)

Reply:
Totally your call either way.  I don't have strong opinions either way here.

Reply:
Thanks

---
In reply to: [338677518](https://github.com/Azure/azure-iot-sdk-node/pull/646#discussion_r338677518) [](ancestors = 338677518)